proxywasm/hostcall.go

```diff
@@ -50,6 +50,14 @@ func GetCurrentTime() int64 {
 	return t
 }
 
+func GetRootContextByID(rootContextID uint32) (RootContext, error) {
```

Comment:
I don't think this file is not the right place to put this function. I'd prefer `vmstate.go` since this has nothing to do with `hostcall`

Reply:
Good point, my bad. I have moved it to `vmstate.go`.